api/schema.py

```diff
@@ -33,3 +38,41 @@ def _get_response_bodies(self, direction: str = "response") -> dict[str, Any]:
                 status_code="400",
             )
         return bodies
+
+
+class TeacherNonTeachingHelpProvidedFieldFix(OpenApiSerializerFieldExtension):
```

Comment:
Maybe this will change or not be needed at all if we add the choices to the model itself?

Reply:
I don't think so. Choices can help in schema when they show some validation method - to show only some values as acceptable. But here we show all this fields as nested field, so we need to fix schema maually

Reply:
Yes, I figured as much, just wanted to check.